repo/packages/E/elasticsearch/0/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "elasticsearch",
+  "version": "0.1.0",
+  "tags": ["dcos", "mesos", "framework", "elasticsearch", "elastic"],
+  "maintainer": "pnw@trifork.com",
+  "description": "DCOS implementation of the Mesos-Elasticsearch framework",
+  "scm": "https://github.com/mesos/elasticsearch.git",
+  "website": "https://github.com/mesos/elasticsearch",
+  "images": {
+    "icon-large": "https://www.elastic.co/static/img/logo-elastic.png"
```

Comment:
This is our bug :)  but we need images for the small and medium as well.   I just ran into this problem with another framework.